v0/destinations/blueshift/transform.js

```diff
@@ -0,0 +1,114 @@
+const { EventType } = require("../../../constants");
+const {
+  CustomError,
+  constructPayload,
+  ErrorMessage,
+  defaultRequestConfig,
+  defaultPostRequestConfig,
+  getValueFromMessage,
+  getFieldValueFromMessage,
+  removeUndefinedAndNullValues
+} = require("../../util");
+
+const {
+  MAPPING_CONFIG,
+  CONFIG_CATEGORIES,
+  BASE_URL_EU,
+  BASE_URL
+} = require("./config");
+
+function checkValidEventName(str) {
+  if (
+    str.indexOf(".") !== -1 ||
+    str.indexOf(" ") !== -1 ||
+    /[0-9]/.test(str) ||
+    str.length > 64
+  )
+    return true;
+  return false;
+}
+
+const responseBuilderSimple = (message, category, destination) => {
```

Comment:
Let's segregate this into separate functions for Identify, Track, Group. It will be easy in terms of code readability and to classify the cases.

Reply:
Okay, done.